ajax/libs/Modaal/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Modaal",
+  "description": "An accessible dialog window library for all humans.",
+  "version": "0.3.1",
+  "filename": "modaal.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/humaan/Modaal"
+  },
+  "license": "MIT",
+  "keywords": [
+    "modaal"
```

Comment:
Let's add more keywords, such as `modal-plugin`, `jquery-plugin`, `jquery`
These keywords is from https://github.com/humaan/Modaal topics.
Thank you.